### PR TITLE
New API proposal_Subscription Status

### PR DESCRIPTION
**Pull request by chinaunicomyangfan**
#### What type of PR is this?

New API Proposal

#### What this PR does / why we need it:

ShutdownServiceStatus proposal from Open Gateway Drop 4

#### Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

**Comment by eric-murray** [comment]
@camaraproject/api-backlog_codeowners 
Agreed to be merged. Needs approval from at least one codeowner.

**Review by TEF-RicardoSerr — APPROVED** [review]
as per [minutes 08/08](https://wiki.camaraproject.org/display/CAM/2024-08-22+API+backlog+minutes#:~:text=39%20already%20merged.-,34,Issue%20%2334%20can%20be%20closed%20and%20PR%20%2332%20merged,-Closed%20Issues)

**Pull request merged**
